nrf_security/doc/backends.rst

```diff
@@ -11,16 +11,15 @@ The nrf_security module supports multiple enabled backends at the same time.
 This mechanism is intended to extend the available feature set of hardware-accelerated cryptography or to provide alternative implementations of the Mbed TLS APIs.
 Enabling one or more backends adds more configuration options grouped into classes of cryptographic algorithms.
 
-Note that some cryptographic features are provided as an option regardless of the backend enabled.
-In such cases, the feature is compiled using open-source software from the original Arm Mbed TLS project.
-An example of this is `SHA-512`, which is only accessible using open-source software from the Arm Mbed TLS project.
+Note that some cryptographic features are provided by Mbed TLS even if it is disabled.
```

Comment:
We use the statement original Mbed TLS to express the difference between backend support and "builtin". The term builtin is added in PSA driver scope which possibly is better than original. @b-gent can comment on the potential to use this term instead of "original" in this case.

Plus: We add a term that is in use in newer versions of Mbed TLS
Negative: We add a new concept that comes in addition to the "Original Mbed TLS backend"

Reply:
Would 'built-in' replace 'original' in all occurrences or be added as an additional term? I really don't like us juggling with terms here, especially as we already have a mismatch because kconfigs related to this use the term 'vanilla'.

As much as a change of terminology could be confusing, we should probably stick to the terms used in original Mbed TLS docs. So if it's a replacement in all occurrences, I think we should go for it.

Reply:
The term and the configurations noted as such will be called "builtin" (full word). This is not present in any code in the version of mbed TLS that we point to _right now_

We will remove the vanilla configurations and potentially reword with regards to builtin when we change to PSA Crypto APIs as the standard front-end API.

Unfortunately this is still work-in-progress and will not be part of nRF Connect SDK 1.8.0